mmv1/products/netapp/storagePool.yaml

```
@@ -0,0 +1,130 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'storagePool'
+description: |
+  StoragePool is a container for volumes with a service level and capacity.
+  Volumes can be created in a pool of sufficient available capacity. StoragePool capacity is what you are billed for.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'QUICKSTART_TITLE': 'https://cloud.google.com/netapp/volumes/docs/reference/rest/v1/projects.locations.storagePools'
+  api: 'https://cloud.google.com/netapp/volumes/docs/reference/rest/v1/projects.locations.storagePools'
+base_url: projects/{{project}}/locations/{{location}}/storagePools
+self_link: projects/{{project}}/locations/{{location}}/storagePools/{{storage_pool_id}}
+create_url: projects/{{project}}/locations/{{location}}/storagePools?storagePoolId={{storage_pool_id}}
+update_url: projects/{{project}}/locations/{{location}}/storagePools/{{storage_pool_id}}
```

```suggestion
self_link: projects/{{project}}/locations/{{location}}/storagePools/{{name}}
create_url: projects/{{project}}/locations/{{location}}/storagePools?storagePoolId={{name}}
update_url: projects/{{project}}/locations/{{location}}/storagePools/{{name}}
```
We'd prefer to use name refer the short name of the resource. Sorry for missing this in my earlier reviews. I've added commit suggestions in all the places. Hope it helps for updating the change.